### DISCUSSION TITLE
There is a lack of support for some class names, such as `column-count`

### DISCUSSION THREAD

**Discussion by guxuerui** [unocss/unocss#2891] [Q&A] [answered]
I want to use css style such as : `column-count: 2`, but found no support in Unocss,  is my usage correct?

![image](https://github.com/unocss/unocss/assets/23629097/8325f94b-7604-4d44-8d36-11e8c6d21260)

**Comment by zyyv** [comment, marked as answer]
https://unocss.dev/interactive/?s=columns-3

**Reply by guxuerui** [reply]
This solved my problem, thank you !!
